presentations/index.md

```diff
@@ -128,7 +141,8 @@ Red Hat Summit 2014, San Francisco, California, USA
 ### April 14-17, 2014
 
 -   [Red Hat Storage Server Administration Deep
```

Comment:
This was in previous presentations page. We can remove if we'd like, he's done a different version at LinuxCon EU.